.changeset/three-squids-explode.md

```diff
@@ -0,0 +1,12 @@
+---
+'braid-design-system': minor
```

Comment:
This is technically a `major` right? It'll be combined with other `major`s on the `next` branch though, so not fussed.

Reply:
Correct! I had an unsaved change locally for that 🤦 

```suggestion
'braid-design-system': major
```